application/src/main/java/com/github/pieter_groenendijk/model/Copies.java

```diff
@@ -0,0 +1,65 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+
+@Entity
+@Table (name = "Copies")
+public class Copies {
```

Comment:
Ik snap niet helemaal het idee achter deze entity. Hij slaat informatie op over een `PhysicalProduct` en dan specifiek over de kopieën?

Deze tabel is momenteel te vaag voor mij om er concreet iets over te zeggen. In mijn hoofd moet er of per kopie van een fysiek product een record zijn met status enz. Of, het gaat over de gehele groep met `numberOfCopies` bijv.. Ik gok dat laatste?

Reply:
Hoe je het beschrijft in Teams klinkt duidelijk en handig. Echter lijkt de daadwerkelijke entiteit af te wijken van hoe je het daar omschrijft.

Reply:
Yes aangepast, inhoud matched omschrijving nu beter